plugins/importexport/native/filter/NativeXmlSubmissionFileFilter.inc.php

```diff
@@ -282,7 +282,7 @@ function handleRevisionElement($node) {
 						if (($e = $childNode->getAttribute('encoding')) != 'base64') {
 							$deployment->addError(ASSOC_TYPE_SUBMISSION, $submission->getId(), __('plugins.importexport.common.error.unknownEncoding', array('param' => $e)));
 						} else {
-							$content = base64_decode($childNode->textContent, true);
```

Comment:
we don't need the "strict" flag here? if no, should we remove the check below also?

Reply:
I would favour leaving the strict flag on, rather than potentially importing partial content without a failure.